auth/downscoped/initialize_CAB.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package downscopedoverview
+
+// [START auth_downscoping_rules]
+
+import "golang.org/x/oauth2/google/downscope"
+
+// constructCAB shows how to initialize a Credential Access Boundary for downscoping tokens.
+func constructCAB(bucketName string) {
+	// A condition can optionally be provided to further restrict access permissions.
```

Comment:
Can you add the objectPrefix as a param? Otherwise this will look weird side by side the other languages in the docs. Go will be the only language that uses the "profile-picture-" prefix.